content/docs/studio/user-guide/account-management.md

```diff
@@ -5,24 +5,59 @@ Iterative Studio. In the menu that comes up, click on `Profile`. Your user
 profile page will open. This page has multiple sections that are described
 below.
 
+## Profile
+
+Here, you can see your name and profile picture. If you signed up with a
+GitHub.com, GitLab.com or Bitbucket.org account, these details are fetched from
+your connected Git hosting account.
+
+You can edit your name.
+
 ## Account
 
-Here, you can see your avatar, user name and email address. These details are
-fetched from your connected Git account and cannot be changed.
+Here, you can see your username, password and email addresses. If you signed up
+with a GitHub.com, GitLab.com or Bitbucket.org account, the username and email
+address are fetched from your connected Git hosting account.
+
+You can update your username and password.
+
+**Managing email addresses:**
+
+You can add multiple email addresses to a single Studio account. You can login
+to the account with any of your verified email addresses.
+
+One of your email addresses must be designated as primary. This is the address
+to which Studio will send all your account notification emails.
+
+You can change your primary email address by clicking on the `Primary` button
+next to the email address which you want to designate as primary.
+
+You can delete your non-primary email addresses.
 
 ## Git integrations
 
 In this section, you can,
 
-- Connect to additional Git providers. For instance, if you are currently
-  connected only to GitHub, you can connect to GitLab and/or Bitbucket from
-  here. When you connect to the additional Git providers, you may be prompted to
-  Grant access to your account to Iterative Studio. Please grant the required
-  access to enable Iterative Studio to connect to your Git repositories.
-- Disconnect from your GitHub, GitLab or Bitbucket accounts.
-- Configure your GitHub account. That is, install the Iterative Studio GitHub
-  app on additional organizations or repositories, or even remove the app from
-  organizations or repositories where you no longer need it.
+- Connect to GitHub.com, GitLab.com or Bitbucket.org.
+
+  When you connect to a Git provider, you may be prompted to grant access to
+  your account to Iterative Studio. Please grant the required access to enable
+  Iterative Studio to connect to your Git repositories.
+
+  If you signed up to use Studio with an email address, you will not have any of
+  the Git connections when you first sign in to Studio, and you can set up the
+  required Git connections in this section.
+
+  Note that **connections to self-hosted GitLab servers** are not managed in
+  this section. If you want to connect to a self-hosted GitLab server, you
+  should create a team and set up the GitLab server connection in the team
+  settings. For more details, refer
+  [here](/doc/studio/user-guide/connect-custom-gitlab-server).
+
+- Disconnect from your GitHub.com, GitLab.com or Bitbucket.org accounts.
```

Comment:
```suggestion
- Disconnect from your GitHub, GitLab or Bitbucket account.

```

Reply:
There's a reason the `.com/org`s were addded (see previous comments). But I guess the user will understand here, so I've removed then as you suggested. But I didn't change `accounts` to `account`, coz the user can disconnect from multiple Git accounts (each on a different provider).